templates/addons/calico-dual-stack/values.yaml

```diff
@@ -0,0 +1,19 @@
+installation:
+  cni:
+    type: Calico
+    ipam:
+      type: HostLocal
+  calicoNetwork:
+    bgp: Disabled
+    mtu: 1350
+    ipPools:
+    - blockSize: 26
+      cidr: 10.244.0.0/16
```

Comment:
Would there be a situation where a pod CIDR list has multiple IPv4 address blocks? If so, how would we set this field? Would it be the union of the two blocks?

Reply:
I don't think that's possible but let me do more research on that to verify

Reply:
confirmed. CAPI is also adding validation to enforce this: https://github.com/kubernetes-sigs/cluster-api/pull/7420